lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -91,7 +91,10 @@
         </li>
     </ul>
     <div class="submit-email-action">
-        <p class="copy">${_("We recommend sending learners no more than one email message per week. Before you send your email, review the text carefully and send it to yourself first, so that you can preview the formatting and make sure embedded images and links work correctly.")}</p>
+        <p class="copy">
+            ${_("We recommend sending learners no more than one email message per week. Before you send your email, review the text carefully and send it to yourself first, so that you can preview the formatting and make sure embedded images and links work correctly.")}
+            ${_("You can include keywords in your messages. For list of available keywords see ")} <a href="${settings.COURSE_BULK_EMAIL_HELP_URL}" target="_blank" rel="noopener">${_("documentation")}</a>.
```

Comment:
ah one last comment - this isn't correct for internationalization.
```suggestion
            ${_("You can include keywords in your messages. For list of available keywords see the {html_start}documentation{html_end}").format(html_start='<a href="${settings.COURSE_BULK_EMAIL_HELP_URL}" target="_blank" rel="noopener">', html_end='</a>')}
```

I think that works but should be tested. We can't concatenate sentences, that may break languages with different orderings and _always_ breaks RTL languages.